README.md

```diff
@@ -12,24 +12,41 @@ Go to the examples folder and run the following:
 ACCEPT_EULA=<yes/no> docker-compose up --build -d
 ```
 
-Then go to the reload-runner directory and install NodeJs dependencies:
+Then follow the example for either corectl or node.
+
+### Node
+
+Head into [/examples/node](/examples/node) and install the dependencies and then run the script [index.js](/examples/node/index.js) using the following commands:
```

Comment:
I guess the folder should now be named `examples` but it is still named `example` :)